### PR TITLE
docs: add cron job example

### PR DESCRIPTION
**Pull request by benjaminch**
Terraform provider PR for jobs:
https://github.com/Qovery/terraform-provider-qovery/pull/172

### CODE REVIEW

File: examples/deploy-a-cron-job/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+```shell
+export TF_VAR_aws_access_key_id=YOUR_AWS_ACCESS_KEY_ID \
+TF_VAR_aws_secret_access_key=YOUR_AWS_SECRET_ACCESS_KEY \
+TF_VAR_qovery_access_token=YOUR_QOVERY_API_TOKEN \
+TF_VAR_qovery_organization_id=YOUR_QOVERY_ORG_ID
+```
+
+6. Clone my [Echo app](https://github.com/Qovery/terraform-provider-testing.git)
+7. Edit the `main.tf` file and change `https://github.com/Qovery/terraform-provider-testing.git` with yours
```
Comment (ghost) [inline examples/deploy-a-cron-job/README.md]: minor: Maybe specify also to change the branch, just to be fully explicit

File: examples/deploy-a-cron-job/main.tf [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    }
+  }
+  
+  source = {
+    docker = {
+      dockerfile_path = "Dockerfile"
+      git_repository = {
+        url = "https://github.com/Qovery/terraform-provider-testing.git"
+        branch = "job-echo-n-seconds"
+        root_path = null
```
Comment (ghost) [inline examples/deploy-a-cron-job/main.tf]: Maybe put the root_path explicitly to `/` (this is the value set by the core if null)

Comment (benjaminch) [reply examples/deploy-a-cron-job/main.tf]: 👍

File: examples/deploy-a-cron-job/README.md (hunk 1 above)
Comment (benjaminch) [reply examples/deploy-a-cron-job/README.md]: 👍

Comment (benjaminch) [reply examples/deploy-a-cron-job/README.md]: Fixed

File: examples/deploy-a-cron-job/main.tf (hunk 2 above)
Comment (benjaminch) [reply examples/deploy-a-cron-job/main.tf]: Fixed

**Review by ghost — APPROVED** [review]

**Pull request merged**